glide.yaml

```diff
@@ -28,7 +28,7 @@ import:
     version: ^0.8
 
   - package: github.com/apache/thrift
-    version: 0.9.3-pool-read-binary-2
+    version: 0.9.3-pool-read-binary-3
```

Comment:
Can we just merge this into master of our branch? Pretty confusing that we're pinned to a branch

Reply:
Not opposed to it, problem is master is way ahead of where we are at, so we'd have to reset master all the way back to 0.9.3 on our fork which would be weird...